Install phase prints this, but exits with code 0. Is this OK?
```
create-icons.pl
Can't locate Graphics/Magick.pm in @INC (you may need to install the Graphics::Magick module) (@INC contains: /home/uj/.opam/4.02.3/lib/perl5 /home/uj/ya/qabs-trunk/qabs-base-modules/ /home/uj/.emacs.d/pieces/perl/sepia/lib /nix/store/yj4b630bm6yscdw8j2sa1sb61j3rdqsy-perl-5.24.2/lib/perl5/site_perl/5.24.2/x86_64-linux-thread-multi /nix/store/yj4b630bm6yscdw8j2sa1sb61j3rdqsy-perl-5.24.2/lib/perl5/site_perl/5.24.2 /nix/store/yj4b630bm6yscdw8j2sa1sb61j3rdqsy-perl-5.24.2/lib/perl5/site_perl /nix/store/yj4b630bm6yscdw8j2sa1sb61j3rdqsy-perl-5.24.2/lib/perl5/site_perl/5.24.2/x86_64-linux-thread-multi /nix/store/yj4b630bm6yscdw8j2sa1sb61j3rdqsy-perl-5.24.2/lib/perl5/site_perl/5.24.2 /nix/store/yj4b630bm6yscdw8j2sa1sb61j3rdqsy-perl-5.24.2/lib/perl5/5.24.2/x86_64-linux-thread-multi /nix/store/yj4b630bm6yscdw8j2sa1sb61j3rdqsy-perl-5.24.2/lib/perl5/5.24.2 .) at create-icons.pl line 23.
BEGIN failed--compilation aborted at create-icons.pl line 23.
make[1]: *** [Makefile:156: www/routino/icons/ball-0.png] Error 2
```

--- a/pkgs/tools/misc/routino/default.nix
+++ b/pkgs/tools/misc/routino/default.nix
@@ -1,0 +1,24 @@
+{ stdenv, fetchurl, perl, zlib, bzip2 }:
+
+stdenv.mkDerivation rec {
+  name = "Routino-${version}";
+  version = "3.2";
+
+  src = fetchurl {
+    url = "http://routino.org/download/routino-${version}.tgz";
+    sha256 = "0lkmpi8gn7qf40cx93jcp7nxa9dfwi1d6rjrhcqbdymszzm33972";
+  };
+
+  nativeBuildInputs = [ perl ];
+  buildInputs = [ zlib bzip2 ];
+
+  installFlags = [ "prefix=$(out)" ];
+
+  meta = with stdenv.lib; {
+    homepage = http://www.routino.org/;
+    description = "OpenStreetMap Routing Software";
+    license = licenses.agpl3;
+    maintainter = with maintainers; [ dotlambda ];
+    platforms = with platforms; linux;
+  };
+}

--- a/pkgs/tools/misc/routino/default.nix
+++ b/pkgs/tools/misc/routino/default.nix
@@ -1,18 +1,21 @@
 { stdenv, fetchurl, perl, zlib, bzip2 }:
 
 stdenv.mkDerivation rec {
-  name = "Routino-${version}";
+  name = "routino-${version}";
   version = "3.2";
 
   src = fetchurl {
-    url = "http://routino.org/download/routino-${version}.tgz";
+    url = "http://routino.org/download/${name}.tgz";
     sha256 = "0lkmpi8gn7qf40cx93jcp7nxa9dfwi1d6rjrhcqbdymszzm33972";
   };
 
   nativeBuildInputs = [ perl ];
+
   buildInputs = [ zlib bzip2 ];
 
-  installFlags = [ "prefix=$(out)" ];
+  outputs = [ "out" "doc" ];
+
+  makeFlags = [ "prefix=$(out)" ];
 
   meta = with stdenv.lib; {
     homepage = http://www.routino.org/;

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -4179,6 +4179,8 @@ with pkgs;
 
   rnv = callPackage ../tools/text/xml/rnv { };
 
+  routino = callPackage ../tools/misc/routino { };
+
   rq = callPackage ../development/tools/rq {
     v8 = v8_static;
   };

